qiskit/opflow/gradients/derivative_base.py

```diff
@@ -31,6 +31,13 @@
 
 OperatorType = Union[StateFn, PrimitiveOp, ListOp]
 
+try:
```

Comment:
the linter may catch this...

Reply:
Oops, fixed in: https://github.com/Qiskit/qiskit-terra/pull/6270/commits/d193f29eb2e5ea77e0f4b561f4be6120b9cca862